aten/src/ATen/native/transformers/cuda/attention_backward.cu

```diff
@@ -94,7 +95,8 @@ std::tuple<Tensor, Tensor, Tensor> _flash_attention_backward(
   benchmarking. We will hard code it to 0 for now
   */
   constexpr int num_splits{0};
-  auto softmax_scale = std::pow(query.size(-1), -0.5);
+  const auto softmax_scale =
+      scale.has_value() ? scale.value() : std::pow(query.size(-1), -0.5);
```

Comment:
Here it's `query.size(-1)` and below it's `query.size(3)`. It might seem like a nit, but it also feels like a footgun if we (or someone else!) at some point increases the dimensionality of inputs for some reason for some of these kernels and don't update this code.